features/step_definitions/amazon_search.rb

```diff
@@ -0,0 +1,22 @@
+Given("I am on Amazon home page") do
+  navigate_to($app_path)
+end
+
+When("I enter the text {string} on the top search bar") do |search|
+  top_search_bar = $driver.find_element(id: 'twotabsearchtextbox')
+  top_search_bar.send_keys search
+  top_search_bar.submit
+end
+
+When("I select Polycarbonate as Cell Phone Case Material") do
+  $driver.find_element(xpath: '//*[@id="p_n_feature_ten_browse-bin/17731940011"]/span/a/div/label/i').click
```

Comment:
You can use directly `click(typeIdentifier, identifier)`  to avoid find_element and click.